File: javaFinal/src/javaFinal/Topping.java
package javaFinal;

import java.util.ArrayList;
import java.util.List;

public class Topping {
	
	private List<String> toppings;
	
	public Topping() {
		toppings = new ArrayList<String>();
	}
	
	public void addTopping(String topping) {
		toppings.add(topping);
	}
	
	public void display() {
		System.out.println("The ice cream has these toppings:");
		for (String t : toppings) {
			System.out.println(t);
		}
	}
}

File: cppFinal/Topping.cpp
#include "Topping.h"
#include "iostream"
using namespace std;

Topping::Topping() {
    
}

void Topping::addTopping(string topping) {
    toppings.push_back(topping);
}

void Topping::display() {
    cout << "This ice cream has these toppings: " << endl;
    for(string t : toppings) {
        cout << t << endl;
    }
}

Topping::~Topping() {
    
}
